source/stylesheets/base/_feature_sections.scss

```diff
@@ -21,32 +21,44 @@
 
     .illustration {
       align-items: flex-end;
-      justify-content: center;
+      justify-content: flex-end;
     }
 
     .text-and-description {
       @include grid-column(6 of 12);
       align-items: center;
       display: flex;
     }
+
+    h1 {
+      font-size: 3.1em;
```

Comment:
I'd try and move away from relying on elements in your components. If someone comes in later and changes this to an `h2` it will break. What do you think about using a class name like, `.jumbo-feature-headline`, or something along those lines?

Reply:
I'm working in another branch to turn this into a partial so I can reuse it similarly on the individual recipe pages, so I'll address this there during that process.